src/guessWords/containers/Input.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, Fragment, useCallback } from "react";
+
+function Input(props) {
+  const [guessedWord, setGuessedWord] = useState("");
+  const onChangeCallback = useCallback((event) => {
+    setGuessedWord(event.target.value);
+  });
+  const onSubmitGuessedWord = () => {
```

Comment:
use `useCallback` for this function.

Reply:
instead of using callbacks, I tried converting it to non-inline form